jetson/nucleo_bridge/README.md

```diff
@@ -0,0 +1,104 @@
+
+#### SA Closed Loop \[Subscriber\] "/sa_closedloop_cmd"
+Message: [SAClosedLoopCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/SAClosedLoopCmd.lcm) \
+Publisher: onboard/kinematics \
+Subscriber: onboard/nucleo_bridge
+
+#### Gimbal Open Loop \[Subscriber\] "/gimbal_openloop_cmd"
+Message: [GimbalCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/GimbalCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### Hand Open Loop \[Subscriber\] "/hand_openloop_cmd"
+Message: [HandCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/HandCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### Foot Open Loop \[Subscriber\] "/foot_openloop_cmd"
+Message: [FootCmd.lcm](https://github.com/raytitan/mrover-workspace/blob/rnucleo/rover_msgs/FootCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### RA Pos Data \[Publisher\] "/arm_posiiton"
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
+Publisher: onboard/nucleo_bridge \
+Subscriber: onboard/kinematics
+
+#### SA Pos Data \[Publisher\] "/sa_pos_data"
+Message: [SAPosData.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/SAPosData.lcm) \
+Publisher: onboard/nucleo_bridge \
+Subscriber: onboard/kinematics
+
+### Usage
+
+To build nucleo_bridge use `$./jarvis build onboard/nucleo_bridge/ ` from the mrover-workspace directory.
+
+To run nucleo_bridge use `$./jarvis exec onboard/nucleo_bridge/ `
+from the mrover-workspace directory.
+
+After initializing the LCM bus, I2C bus, and virtual Controller objects, the CLI will only show errors, since printing output to the console is time expensive. A blank CLI is a good thing.
+
+To control the RA/SA through open-loop
+* Ensure onboard/teleop is running on the same platform
+* Ensure base_station/gui is running on the base station
+* Operate the joysticks with the RA/SA Task page open on the base station GUI
+
+To control the RA/SA through closed-loop
+* Ensure onboard/teleop is running on the same platform
+* Ensure onboard/kinematics is running on the same platform
+* Ensure base_station/kineval is running on the base station
+* Input commands into the arm control page on the base station GUI
+
+### Common Errors
+
+This routine typically only thows one type of error, when it has issues communicating with the Nucleo motor controllers. They will have the form "command failed on channel"
+
+The Nucleo that is unresponsive is the first digit of the channel minus 1. For example, if the error message says "activate failed on 10", check Nucleo 0. Typically this is because the wiring on the i2c bus has failed nd needs to be fixed. Even after wiring is fixed or other related issues have been resolved, the Nucleos,To reset the Nucleo, press it's power button. If this does not fix nucleo_bridge, you may have to restart nucleo_bridge.

```

Comment:
active failed? i thought the error msg was command failed on channel _

Reply:
if the message is active failed --> put that as the header name ('Active Failed XY) , and the description of what it means/how to fix it under

Reply:
"command" is a generic command, it could be any of the transaction types between the motor nucleo firmware and bridge.

The common fix for all of these errors is to fix the i2c bus, since transactions don't fail unless a motor nucleo is not responding, or we screwed up a major part of our transaction code (which likely wouldn't be fixed on-the-fly).

Reply:
okay makes sense, just make sure that is communicated in the documentation following the readme guide